py3status/modules/air_quality.py

```diff
@@ -0,0 +1,190 @@
+    color_unhealthy: Unhealthy (default '#CC0033')
+    color_unhealthy_sensitive: Unhealthy for Sensitive Groups (default '#FF9933')
+    color_unknown: Unknown (default '#FFFFFF')
+    color_very_unhealthy: Very Unhealthy (default '#660099')
+    format: display format for this module (default 'AQI: {aqi} {category}')
+    icon_good: (default 'Good')
+    icon_hazardous: (default 'Hazardous')
+    icon_moderate: (default 'Moderate')
+    icon_unhealthy: (default 'Unhealthy')
+    icon_unhealthy_sensitive: (default 'Unhealthy for Sensitive Groups')
+    icon_unknown: (default 'Unknown')
+    icon_very_unhealthy: (default 'Very Unhealthy')
+    location: location name/uid to query. To search for nearby stations in
+        Kraków, use `curl http://api.waqi.info/search/?token=YOUR_TOKEN&keyword=kraków`
+        We recommend you to use uid instead of name in location, eg "@8691"
+        (default 'Shanghai')
+
+Format placeholders:
+    {aqi} air quality index
+    {category} health risk category
+    {location} location name/uid
+    {random} output from "random" parameter in `iaqi` array.
+
+__Note: Station may have their set of custom parameters such as {pm25}, {pm10}, etc.__
+
+Color options:
+    color_good: Good (default '#009966')
+    color_hazardous: Hazardous  (default '#7E0023')
+    color_moderate: Moderate (default '#FFDE33')
+    color_unhealthy: Unhealthy (default '#CC0033')
+    color_unhealthy_sensitive: Unhealthy for Sensitive Groups (default '#FF9933')
+    color_unknown: Unknown (default '#FFFFFF')
+    color_very_unhealthy: Very Unhealthy (default '#660099')
+
+Requires:
+    python-requests: Python HTTP for Humans
+
+Example:
+```
+air_quality {
+    auth_token = 'demo'
+    location = 'Shanghai'
+    format = 'Shanghai: {aqi} {category}'
+}
+```
+@author beetleman
+@license BSD
+"""
+
+try:
+    # python 3
+    from urllib.error import URLError
+    from urllib.request import urlopen, Request
+    from urllib.parse import urlencode
+except ImportError:
+    # python 2
+    from urllib import urlencode
+    from urllib2 import URLError
+    from urllib2 import urlopen, Request
+import json
+
+AQI = (
+    (0, 50, 'icon_good'),
+    (51, 100, 'icon_moderate'),
+    (101, 150, 'icon_unhealthy_sensitive'),
+    (151, 200, 'icon_unhealthy'),
+    (201, 300, 'icon_very_unhealthy'),
+    (300, None, 'icon_hazardous')
+)
+URL = 'http://api.waqi.info'
+STRING_ERROR = 'air_quality: unknown error'
+
+
+def get_in(coll, path, default=None):
+    first = path[0]
+    rest = path[1:len(path)]
+    if len(path) == 1:
+        return coll.get(first, default)
+    return get_in(coll.get(first, default), rest, default)
+
+
+class Py3status:
+    auth_token = 'demo'
+    cache_timeout = 3600
+    color_good = '#009966'
+    color_hazardous = '#7E0023'
+    color_moderate = '#FFDE33'
+    color_unhealthy = '#CC0033'
+    color_unhealthy_sensitive = '#FF9933'
+    color_unknown = '#FFFFFF'
+    color_very_unhealthy = '#660099'
```

Comment:
these `color_...` attributes should die.

can we just keep the defaults somewhere

```
COLOR_DEFAULTS = {
    'hazardous': '#7E0023',
    'moderate': '#FFDE33',
}
```